element_miniscope/imaging.py

```diff
@@ -83,12 +84,14 @@ class ProcessingParamSet(dj.Lookup):
     param_set_hash: uuid
     unique index (param_set_hash)
     params: longblob  # dictionary of all applicable parameters
+    package_version=null: varchar(16)
```

Comment:
I recall we discussed having `package_version` as secondary attribute at `Processing` table